satpy/composites/spectral.py

```diff
@@ -68,3 +70,54 @@ def __call__(self, projectables, optional_datasets=None, **attrs):
         new_green = sum(fraction * value for fraction, value in zip(self.fractions, projectables))
         new_green.attrs = combine_metadata(*projectables)
         return super(GreenCorrector, self).__call__((new_green,), **attrs)
+
+
+class NDVIHybridGreen(GenericCompositor):
+    """Construct a hybrid green channel weigthed by NDVI.
```

Comment:
```suggestion
    """Construct a hybrid green channel weighted by NDVI.
```

Reply:
Thanks, I have fixed this in the later commits.